src/traffic_cop/traffic_cop.cpp

```diff
@@ -358,6 +357,24 @@ TrafficCopResult TrafficCop::ExecuteExplainStatement(
   std::vector<planner::OutputSchema::Column> output_columns;
   output_columns.emplace_back("QUERY PLAN", type::TypeId::VARCHAR, nullptr);
 
+  // TODO(WAN): Integrate with Matt's PR.
+#if 0
+    // Codegen must happen for certain types of EXPLAIN metadata to be collected, e.g., collection of TPL.
```

Comment:
Integration with #1588, check the FORMAT to decide whether you're dumping the plan node, or codegenning the physical plan and then dumping the compile time metadata.